documentation/docs/admin/security.mdx

```diff
@@ -80,3 +80,36 @@ sudo systemctl restart nginx
 <Callout variant={CalloutVariant.info}>
     Ensure all nodes use the same SSH key set and restrict access to configuration files for system users only.
 </Callout>
+
+## Rate Limiting
```

Comment:
Thanks for directly adding this in the new documentation 🙏 

Can we just add a link to this section in the old documentation (as it is not migrated yet and will most likely take some weeks / months)